flutter/lib/common/widgets/setting_widgets.dart

```diff
@@ -212,6 +212,8 @@ List<(String, String)> otherDefaultSettings() {
     if ((isDesktop || isWebDesktop)) ('show_monitors_tip', kKeyShowMonitorsToolbar),
     if ((isDesktop || isWebDesktop)) ('Collapse toolbar', 'collapse_toolbar'),
     ('Show remote cursor', 'show_remote_cursor'),
+    ('Follow remote cursor', 'follow_remote_cursor'),
+    ('Follow remote window focus', 'follow_remote_window'),
```

Comment:
Enabling these two default settings, but not "Show Remote Cursor", will not send the show remote cursor the first time you connect.